proto/umee/incentive/v1/genesis.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive";
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "umee/incentive/v1/incentive.proto";
+
+// GenesisState defines the x/incentive module's genesis state.
+message GenesisState {
+  Params                    params               = 1 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram completed_programs   = 2 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram ongoing_programs     = 3 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram upcoming_programs    = 4 [(gogoproto.nullable) = false];
+  uint32                    next_program_id      = 5;
+  uint64                    last_rewards_time    = 6;
+  repeated cosmos.base.v1beta1.Coin total_bonded = 7 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  repeated BondAmount        bond_amounts        = 8 [(gogoproto.nullable) = false];
+  repeated PendingReward     pending_rewards     = 9 [(gogoproto.nullable) = false];
+  repeated RewardBasis       reward_bases        = 10 [(gogoproto.nullable) = false];
+  repeated RewardAccumulator reward_accumulators = 11 [(gogoproto.nullable) = false];
+  repeated Unbonding         unbondings          = 12 [(gogoproto.nullable) = false];
+}
+
+// BondAmount tracks the amount of coins of one uToken denomination bonded to a
+// given reward tier by a single account.
```

Comment:
```suggestion
// Bond tracks the user rewards per tire and uToken coin.
```

Reply:
I think we can simplify the name, otherwise we would need to call it `UserTireBond` or `UserTireCoinBond`